thyrel-web/src/pages/ComponentTest.tsx

```diff
@@ -124,6 +126,14 @@ export default function ComponentTest() {
           directive="Time to draw"
           sentence="Mémé fait des fucks à la police"
         />
+
+        <SizePicker
+          currentSize={size}
+          onSizeChange={size => {
+            console.log('Size is :', size);
+            setSize(size);
+          }}
```

Comment:
```suggestion
          onSizeChange={setSize}
```
c'est pour tester que je t'avais fais changer, mais c'est + propre comme ça

après osef car c'est un composant test